scripts/start-utils

```diff
@@ -1,5 +1,23 @@
 #!/bin/bash
 
+function get_from_gh() {
```

Comment:
It would be really cool to port this over to 

https://github.com/itzg/mc-image-helper/blob/master/src/main/java/me/itzg/helpers/get/GetCommand.java

and/or a `verify-gh-scope` subcommand.